packages/common-server/src/filesv2.ts

```diff
@@ -569,6 +569,26 @@ export async function findNonNoteFile(opts: {
 }
 
 class FileUtils {
+  /**
+   * Keep incrementing a numerical suffix until we find a path name that does not correspond to an existing file
+   * @param param0
+   */
+  static genFilePathWithSuffixThatDoesNotExist({
+    fpath,
+    sep = "-",
+  }: {
+    fpath: string;
+    sep?: string;
+  }) {
+    // Try to put into a eefault '~/Dendron' folder first. If path is occupied, create a new folder with an numbered suffix
```

Comment:
nit: this function doesn't specify anything about a `~/Dendron` folder.  Also, sp on 'default'